### DISCUSSION TITLE
Help on doing a project for c programming with Vs code

### DISCUSSION THREAD

**Discussion by SsemuliJoseph** [microsoft/vscode-cpptools#12676] [General]
Help on doing a project for c programming with Vs code

**Comment by sean-mcmanus** [comment]
@Ssemuli You haven't provided enough information.

**Reply by SsemuliJoseph** [reply]
Hello. I'm currently a first year computer science student. Help on what to do to be competent enough. And advice on choosing the right path in computer science. Thanks

**Comment by SsemuliJoseph** [comment]
I am a student doing computer science. I wanted to know how to use Vs code
to correctly learn c programming with the correct syntax because I have
been using YouTube videos and practicing with Vs code. But I constantly get
error codes even when I do the same procedure as portrayed in the video.
Thanks
